index.html

```diff
@@ -0,0 +1,117 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>판다마켓</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <!-- 헤더 -->
+    <header class="header">
+        <div class="header-content">
+            <a href="/">
+                <img class="logoImg" src="/images/logo.png" alt="logo">
+            </a>
+           
+            <a href="/pages/login.html" class="header-button button">로그인</a>
+        </div>
+    </header>
+ <!-- 상단랜딩 -->
+    <div class="topLanding">
+        <div class="top-content">
+            <div class="top-content-text">
+                <h2>일상의 모든 물건을<br>거래해 보세요</h2>
+                <a href="/pages/items.html" class="items-button button">구경하러 가기</a>
+
+            </div>
+           
+            <img  class="topLandingImg" src="/images/topImg.png" alt="topImg">
+
+        </div>
+
+    </div>
+<!-- 섹션 -->
+    <section class="section">
+
+        <div class="section-content">
+            <div class="section-card">
+                <img  class="sectionImg" src="/images/card1.png" alt="sectionImg1">
+                <div class="section-text card1">
+                    <p class="section-text-top">Hot item</p>
+                    <p class="section-text-middle">인기 상품을<br>확인해 보세요</p>
+                    <p class="section-text-bottom">가장 HOT한 중고거래 물품을<br>판다 마켓에서 확인해 보세요</p>
+                </div>
+                
+            </div>
+        </div>
+
+        <div class="section-content">
+            <div class="section-card">
+                <div class="section-text card2">
+                    <p class="section-text-top">Search</p>
+                    <p class="section-text-middle">구매를 원하는<br>상품을 검색하세요</p>
+                    <p class="section-text-bottom">구매하고 싶은 물품은 검색해서<br>쉽게 찾아보세요</p>
+                </div>
+
+                <img  class="sectionImg" src="/images/card2.png"alt="sectionImg2" >
+                
+            </div>
+        </div>
+
+        <div class="section-content">
+            <div class="section-card card3">
+                <img  class="sectionImg" src="/images/card3.png" alt="sectionImg3" >
+                <div class="section-text">
+                    <p class="section-text-top">Register</p>
+                    <p class="section-text-middle">판매를 원하는<br>상품을 등록하세요</p>
+                    <p class="section-text-bottom">어떤 물건이든 판매하고싶은 상품을<br>쉽게 등록하세요</p>
```

Comment:
💊 제안

classname을 지으실때 위치에 대한 이름은 추천드리지 않습니다.
나중에 디자인이 변해 태그의 순서가 변했을때, 혹은 반응형 사이트라 화면 크기에 따라 태그 위치가 변할 수 있기 때문에
해댱 요소를 잘 설명하지 못하기 때문입니다.
따라서 가능하면 해당 태그의 역할이나 내용을 설명하는 이름을 사용하는 것이 좋습니다.

```suggestion
                    <p class="section-text-tag">Register</p>
                    <p class="section-text-title">판매를 원하는<br>상품을 등록하세요</p>
                    <p class="section-text-subTitle">어떤 물건이든 판매하고싶은 상품을<br>쉽게 등록하세요</p>
```

이렇게 클래스명을 작성하면 이름을 통해 태그의 역할을 추측할 수 있게되고 위치가 바뀌더라도 코드를 더 잘 이해할 수 있습니다.